src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -1341,37 +1339,31 @@ public enum Dataset
         /// <summary>
         /// The options for the <see cref="ImageClassificationTransformer"/>.
         /// </summary>
-        internal sealed class Options : TransformInputBase
+        public sealed class Options
         {
-            /// <summary>
-            /// Location of the TensorFlow model.
-            /// </summary>
-            [Argument(ArgumentType.Required, HelpText = "TensorFlow model used by the transform. Please see https://www.tensorflow.org/mobile/prepare_models for more details.", SortOrder = 0)]
-            public string ModelLocation;
-
             /// <summary>
             /// The names of the model inputs.
             /// </summary>
-            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The names of the model inputs", ShortName = "inputs", SortOrder = 1)]
-            public string[] InputColumns;
+            [Argument(ArgumentType.Multiple , HelpText = "The names of the model inputs", ShortName = "inputs", SortOrder = 1)]
+            internal string[] InputColumns;
```

Comment:
> internal string[] InputColumns; [](start = 11, length = 32)

probably don't need this or Output coliumns but I will remove this in my PR where I will refactor all this code and convert to ITrainerEstimator